specification/trafficmanager/resource-manager/Microsoft.Network/stable/2018-04-01/trafficmanager.json

```diff
@@ -0,0 +1,1578 @@
+            "description": "Default response. It will be deserialized as per the Error definition.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Profile-DELETE": {
+            "$ref": "./examples/Profile-DELETE.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Profiles"
+        ],
+        "operationId": "Profiles_Update",
+        "description": "Update a Traffic Manager profile.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group containing the Traffic Manager profile."
+          },
+          {
+            "name": "profileName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Traffic Manager profile."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Profile"
+            },
+            "description": "The Traffic Manager profile parameters supplied to the Update operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The updated Traffic Manager profile.",
+            "schema": {
+              "$ref": "#/definitions/Profile"
+            }
+          },
+          "default": {
+            "description": "Default response. It will be deserialized as per the Error definition.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Profile-PATCH-MonitorConfig": {
+            "$ref": "./examples/Profile-PATCH-MonitorConfig.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Network/trafficManagerGeographicHierarchies/default": {
+      "get": {
+        "tags": [
+          "GeographicHierarchies"
+        ],
+        "operationId": "GeographicHierarchies_GetDefault",
+        "description": "Gets the default Geographic Hierarchy used by the Geographic traffic routing method.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The default Geographic Hierarchy.",
+            "schema": {
+              "$ref": "#/definitions/TrafficManagerGeographicHierarchy"
+            }
+          },
+          "default": {
+            "description": "Default response. It will be deserialized as per the Error definition.",
```

Comment:
nit: it is good documentation to specify all the error codes returned by the service. An example [here](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/Microsoft.Sql/stable/2015-05-01/capabilities.json#L49)

Reply:
This is an old operation that hasn't changed for years. Since there are no parameters to verify, there's actually no other expected response than the 200.

Reply:
@allencal you can disregard this